webpack/assets/javascripts/react_app/components/common/table/reducers/createTableReducer.js

```diff
@@ -12,6 +12,7 @@ const initState = Immutable({
 
 const createTableReducer = tableID => (state = initState, action) => {
   const { REQUEST, FAILURE, SUCCESS } = createTableActionTypes(tableID);
+  const { payload, response } = action;
```

Comment:
just a thought - since we're destructuring the action, should we also do that for the `type`? maybe in the method signature even? (also for the rest of the reducers)